docs/tutorial/windows-arm.md

```diff
@@ -0,0 +1,95 @@
+# Windows 10 on Arm
```

Comment:
It should be spelled "Windows 10 on ARM" according to https://docs.microsoft.com/en-us/windows/arm/

Reply:
Ah yes, the "Arm" versus "ARM" thing: it depends on who you ask (https://www.arm.com/company/policies/trademarks/guidelines-trademarks). It might be acceptable in this context (since it's Microsoft's official name for the platform) but I'll double-check with our trademarks team.

Reply:
it's fine